public/static/global.css

```diff
@@ -408,10 +408,11 @@ button:focus {
 .app {
   display: grid;
   min-height: 100vh;
-  max-width: var(--app-width);
+  min-width: var(--app-width);
```

Comment:
we should use `min-width` instead of `max-width` here to make sure that the app width always takes (almost, minus the padding) as much space as the allocated `app-width` values.